### PR TITLE
Use console quick start drawer

### PR DESCRIPTION
**Pull request by batzionb**
Make quick starts open in the console drawer instead of the one embedded in the cluster templates page, this way they stay open also when navigating to other pages.
The fix is done by not wrapping the components with a QuickStart context, this way the console one is used

before

image

after 

image

### CODE REVIEW

File: src/components/ClusterTemplateDetailsPage/ClusterTemplateDetailsPage.tsx [hunk 1]
```diff
@@ -38,7 +38,6 @@ import { getResourceVendor } from '../../utils/clusterTemplateDataUtils';
 import useClusterTemplateActions from '../../hooks/useClusterTemplateActions';
 import useClusterTemplateDeserializer from '../../hooks/useClusterTemplateDeserializer';
 import Alerts from '../../alerts/Alerts';
-import WithClusterTemplateQuickStarts from '../ClusterTemplatesGettingStarted/WithClusterTemplateQuickStarts';
```
Comment (rawagner) [inline src/components/ClusterTemplateDetailsPage/ClusterTemplateDetailsPage.tsx:41]: `WithClusterTemplateQuickStarts` can be removed completely, right ?

Comment (batzionb) [reply src/components/ClusterTemplateDetailsPage/ClusterTemplateDetailsPage.tsx:41]: I use it to generate this component in isolation in storybook
Perhaps I can put it in the story, or in a folder storybookUtils

Comment (batzionb) [reply src/components/ClusterTemplateDetailsPage/ClusterTemplateDetailsPage.tsx:41]: right. deleted.

**Review by rawagner — APPROVED** [review]

**Pull request merged**